### DISCUSSION TITLE
Why does the order of the drop areas vary?

### DISCUSSION THREAD

**Discussion by gingerbeardman** [aonez/Keka#1421] [Q&A]
Every time I drop stuff onto Keka I have to take several seconds to read the screen to figure out what order the drop areas are in. I find it really annoying.

I think it should be consistent.

As it is, I think it's detrimental to the user experience.

For example:

1. dropping a bunch of ISO files

![Screen shot 2024-03-24 at 17 57 17](https://github.com/aonez/Keka/assets/49612/53d46f4b-ff32-48ae-8623-6e35876b1811)

2. dropping a CUE and BIN

![Screen shot 2024-03-24 at 17 57 01](https://github.com/aonez/Keka/assets/49612/a98a8929-b779-4bd1-aea4-cd84056f47cd)

**Comment by aonez** [comment]
It uses the automatic action, depending on the files you’re dropping. ISO files can be extracted so this is the main action, CUE files can’t be extracted so compression will be the main action.

You can set Keka to always compress/extract by default, that way the main drop action will be the same.

Maybe this example is an exception, because the nature of CUE+BIN, and this pair should be treated like one extraction.

**Reply by gingerbeardman** [reply]
I see, it's a hornet's nest :)
